plugins/woocommerce-admin/client/products/layout/product-field-layout.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import { WooProductFieldItem } from './woo-product-field-item';
+
+type ProductFieldLayoutProps = {
+	name: string;
+};
+
+export const ProductFieldLayout: React.FC< ProductFieldLayoutProps > = ( {
+	name,
+	children,
+} ) => {
+	return (
+		<div className="product-field-layout">
+			<WooProductFieldItem.Slot fieldName={ name } location="before" />
```

Comment:
How do multiple fills for a slot get ordered?

Reply:
First come first serve at the moment, I can add the order part back in that was also used in the `WooHeaderItem` example.